Add documentation for Pulumi registry (#27)
1Password/pulumi-onepassword: docs/_index.md

---
title: 1Password
meta_desc: Provides an overview of the 1Password Provider for Pulumi.
layout: package
---

The Aiven provider for Pulumi can be used to provision any of the cloud resources available in [Aiven](https://aiven.io/).

> The Aiven provider must be configured with credentials to deploy and update resources in Aiven.

Reviewer: ```suggestion
You'll need to configure the Aiven provider with credentials to deploy and update resources in Aiven.
```

Could we rephrase like this to avoid passive voice here?

Author: For sure. Also, I've noticed that I forgot to change these to mention the 1Password provider instead of Aiven. 🤦🏻
